.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
           - dragonball
           - qemu
           - stratovirt
+          - cloud-hypervisor
```

Comment:
@GabyCT What's the difference between cloud-hypervisor and clh in this matrix?

Reply:
@sprt - `clh` refers to the golang runtime (the current default) configuration for Cloud Hypervisor, whereas `cloud-hypervisor` refers to the rust runtime (runtime-rs) configuration for Cloud Hypervisor. The names are different since they refer to different config files for each runtime since the runtimes are  very slightly different externally.

We could try to make both config formats the same, but that could end up causing a lot of confusion, particularly when we start thinking about changing the default Kata runtime, so the current view is that having separate configuration files for each runtime will actually help minimise confusion (but note this is still open to discussion).

For further information, see #5954 and #6020, and #8483 where we force the separation (for clarity) between the golang and rust runtime config files by requiring them to be in different directories.